style.css

```diff
@@ -274,13 +274,19 @@ article
 	grid-column: 1 / -2;
 	font-size: 1.5rem;
 	letter-spacing: 0.005em;
-	line-height: 1.6;
+	line-height: 1.8;
 	padding: 40px 20px 50px;
 }
 
 article p
 {
-	margin: 20px 0;
+	margin: 40px 0;
+	padding: 0 40px;
+}
+
+article h2, article ul, article ol, article blockquote
```

Comment:
Same goes for all the "text" elements: headers, lists, blockquotes. (`ul` is unordered lists, `ol` is ordered lists). I've prefixed them all with `article`, so it should only effect elements inside of the `<article>` tags.